apps/tsconfig.json

```diff
@@ -1,33 +1,14 @@
+// This tsconfig is used for development. It extends tsconfig.build.json, used for building apps.
```

Comment:
For my understanding - where are we telling the build system to use this file instead of tsconfig.build.json in development?

Also - would it make sense to name this something like `tsconfig.dev.json`?

Reply:
I'll make the comment more clear: this is used by your editor (such as VS Code) to make sure Intellisense works for js and ts files when working in ts. I kept it as `tsconfig.json` to avoid any friction with IDEs detecting this file.

Reply:
Second @molly-moen here, we want to use tsconfig.json just because its the standard.